### PR TITLE
Store patient race and ethnicity in FHIR

### PR DESCRIPTION
**Pull request by axn9t9**
## :floppy_disk: [Code Check]
_Make sure you've checked/done the following before making this PR!_
* Does your code build/run?
* Does your design approach make sense?
* Does your work achieve the intention of the card?
* Does your code have any unintentional side effects -- have you tested a full workflow & checked the logic branches you touched?
* Did you make a unit test for your code (as appropriate)?
* Is your branch relatively up to date with the Development branch (within ~2 commits)
* Have you moved the card to "Needs QA" in Jira

## :flipper: [JIRA Sprint Card]

https://mass-immunization-system.atlassian.net/browse/MVS-693

## :newspaper: [Work Description]

Store race and ethnicity in FHIR resource using US Core Extension

## :vertical_traffic_light: [Testing/QA Notes]

Register patient with registration app
Access patient resource through HAPI server
- Go to localhost:3000/Patient/_QR\_VALUE_ to view resource id
- Go to localhost:8080/hapi-fhir-jpaserver/fhir/Patient/_RESOURCE\_ID_

Verify race and ethnicity appear in resource

### CODE REVIEW

File: broker/models/Patient.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ Patient {
 }
 */
 
+exports.suffixEnums = {
```
Comment (rrucker3) [inline broker/models/Patient.js:37]: HL7 has the suffix as a string, so I don't think we need to validate it other than that. This leaves out stuff like "PhD".

File: broker/endpoints/Patient.js [hunk 2]
```diff
@@ -135,8 +135,12 @@ exports.create = [
     return Patient.genderEnums[genderVal];
   }),
   body('Patient.*.birthDate', 'Please enter patient birth date').trim().isDate({format: 'MM/DD/YYYY'}),
-  body('Patient.*.race', 'Please enter patient race').trim().escape().isLength({min: 1}),
-  body('Patient.*.ethnicity', 'Please enter patient ethnicity').trim().escape().isLength({min: 1}),
+  body('Patient.*.race', 'Please enter patient race').trim().escape().custom((raceVal) => {
```
Comment (rrucker3) [inline broker/endpoints/Patient.js:138]: Do we have to require race and ethnicity?

Comment (axn9t9) [reply broker/endpoints/Patient.js:138]: Race and ethnicity are required fields by the registration app.

File: broker/models/Patient.js (hunk 1 above)
Comment (axn9t9) [reply broker/models/Patient.js:37]: Yes, HL7 accepts any value for suffix, but the registration app only allows these specific values.

**Review by rrucker3 — APPROVED** [review]

**Pull request merged**